src/main/java/org/dinosaur/foodbowl/domain/bookmark/repository/BookmarkRepository.java

```diff
@@ -18,6 +18,9 @@ public interface BookmarkRepository extends Repository<Bookmark, Long> {
     @EntityGraph(attributePaths = {"post", "post.thumbnail"})
     Page<Bookmark> findAllByMember(Member member, Pageable pageable);
 
+    @EntityGraph(attributePaths = {"post", "post.store"})
+    List<Bookmark> findWithPostAndStoreAllByMember(Member member);
```

Comment:
```suggestion
    List<Bookmark> findAllWithPostAndStoreByMember(Member member);
```

통일성 있게 findAllXXX는 어떨까요?!

Reply:
`find~All` 사이에만 추가적인 의미를 표현할 수 있는줄 알았는데 `All~By` 사이에도 추가적인 의미를 작성할 수 있네요!

새롭게 배워갑니다 😀 훨씬 직관적인 표현이 되는 것 같아요 좋습니다 👍